coral/src/app/features/connectors/details/ConnectorDetails.tsx

```diff
@@ -91,6 +144,24 @@ function ConnectorDetails(props: ConnectorOverviewProps) {
         hasPendingRequest={Boolean(connectorData?.connectorInfo.hasOpenRequest)}
       />
 
+      {connectorData?.connectorInfo !== undefined &&
+        !connectorData.connectorInfo.connectorOwner && (
+          <Box marginBottom={"l1"}>
+            <ClaimBanner
+              entityType={"connector"}
+              entityName={connectorName}
+              hasOpenClaimRequest={
+                connectorData?.connectorInfo.hasOpenClaimRequest
+              }
+              entityOwner={connectorData.connectorInfo.teamName}
+              hasOpenRequest={connectorData?.connectorInfo.hasOpenRequest}
```

Comment:
I think the `ClaimBanner` has a blind spot regarding `hasOpenRequest`. It behaves incorrectly in this case:
- an entity with `hasOpenRequest: true`
- but the request has *not* been opened by your team
- will have a link to an empty table, because we don't see other team's requests in "My team's requests" (sensibly haha)

https://github.com/Aiven-Open/klaw/assets/20607294/43f85c13-4e94-4100-bbe4-36c07bd6582b

It's a little tricky, because it puts into light the fact that there are requests which are completely hidden from users. This is a desirable thing, I think, and part of Klaw's features. But currently `hasOpenRequest` on `connectorInfo` does not guarantee that the request was opened by the current user's team :o

Reply:
is this happening with `hasOpenRequest`? Or with `hasOpenClaimRequest`? Because `hasOpenRequest` will not show a link, only `hasOpenClaimRequest` does. 

In that case, yes, if the claim request is not from the team you're logged in, it's currently a bit weird, we already have seen that with `topics` and decided to (for now) take it as it is and see if we add a property later to differ between "This claim request is from your team" or "you should not see anything here" (or handle it in backend and only return `hasOpenClaimRequest` as true if it's the same team).

(tagging @aindriu-aiven too)

Reply:
Yes sorry I confused the two things, this is when `hasOpenClaimRequest` is `true`.

And yes, maybe if the claim request is opened by another team, then it could be treated as a generic opene request, not specifically a claim request. 🤔

Reply:
💡  ok but then it's the known thing. We should create a follow up. Currently it matches the behaviour that topics and schema also show.

Reply:
Yes, I think it is also probably not trivial for the backend. But it's a behaviour we should keep an eye on! Will you create the issue or should I ?

Reply:
I opened it ^^ https://github.com/Aiven-Open/klaw/issues/1722